src/Tools/NoiseDeCorr.jl

```diff
@@ -0,0 +1,36 @@
+export decorrelateSenseMaps, covariance
+
+"""
+        decorrelateSenseMaps(L_inv, senseMaps, numSl)
+
+    multiplies the senseMaps by the noise uncorrelation matrix (L_inv).
+"""
+
+function decorrelateSenseMaps(L_inv::Union{LowerTriangular{Complex{T}, Matrix{Complex{T}}}, Nothing},
+                    senseMaps::Array{Complex{T},4},
+                    numChan::Int64) where {T}
+    if isnothing(L_inv)
+        senseMapsUnCorr = senseMaps
+    else
+        sizeMaps = size(senseMaps)
+        senseMapsUnCorr = reshape(senseMaps, :, numChan) * L_inv'
+        senseMapsUnCorr = reshape(senseMapsUnCorr, sizeMaps)
+    end
+
+    return senseMapsUnCorr
+end
+
+
+"""
+        covariance(noiseData)
+
```

Comment:
```suggestion
The `noiseData` array should be number of noise samples by number of coils.
```

docstring should clarify how `noiseData` is organized - perhaps # samples by # of coils?